pkgs/development/compilers/gcc/default.nix

```
@@ -3,7 +3,7 @@
 , langAda ? false
 , langObjC ? stdenv.targetPlatform.isDarwin
 , langObjCpp ? stdenv.targetPlatform.isDarwin
-, langD ? false
+, langD ? false, gdc ? null
```

```suggestion
, langD ? false, gdc-bootstrap
```

1. I'm not sure what the purpose of the `? null` is since `gdc` is in `all-packages.nix`.
2. It's probably better to call the parameter `gdc-bootstrap`, like `gnat` does.  That way if somebody forgets to provide it explicitly or via an override, the failure mode is "missing parameter `gdc-bootstrap`" instead of having to hunt down some "`infinite recursion detected`".